cohttp.install

```diff
@@ -1,7 +1,16 @@
 bin: [
-  "?cohttp-server-async"
-  "?cohttp-server-lwt"
-  "?cohttp-curl-async"
-  "?cohttp-curl-lwt"
-  "?cohttp-proxy-lwt"
+  "?cohttp_server_async.native" { "cohttp-server-async" }
+  "?cohttp_server_async.byte"   { "cohttp-server-async" }
```

Comment:
(though personally I don't find that relying on this property is very clean, that's the reason why .install files should really be generated by the build system).